Steps4Impact/App/AppController.swift

```diff
@@ -213,6 +214,8 @@ class AppController {
     AKFCausesService.performAPIHealthCheck { (result) in
       switch result {
       case .failed:
+        let loginManager = LoginManager()
```

Comment:
this is technically a part of another bug but that's fine - just leave it in the comments of a PR